src/main/resources/listLookup.json

```diff
@@ -0,0 +1,86 @@
+{
+  "SJP_PUBLIC_LIST": {
+    "friendlyName": "Single Justice Procedure Public List",
+    "shortenedFriendlyName": "SJP Public List",
+    "url": "sjp-public-list",
+    "jurisdictions": ["Single Justice Procedure"],
+    "restrictedProvenances":  []
+  },
+  "SJP_PRESS_LIST": {
+    "friendlyName": "Single Justice Procedure Press List",
+    "shortenedFriendlyName": "SJP Press List",
+    "url": "sjp-press-list",
+    "jurisdictions": ["Single Justice Procedure"],
+    "restrictedProvenances":  ["PI_AAD"]
+  },
+  "CROWN_DAILY_LIST": {
+    "friendlyName": "Crown Daily List",
+    "shortenedFriendlyName": "Crown Daily List",
+    "url": "",
+    "jurisdictions": ["Crown"],
+    "restrictedProvenances":  []
+  },
+  "CROWN_WARNED_LIST": {
+    "friendlyName": "Crown Warned List",
+    "shortenedFriendlyName": "Crown Warned List",
+    "url": "",
+    "jurisdictions": ["Crown"],
+    "restrictedProvenances":  []
+  },
+  "CROWN_FIRM_LIST": {
+    "friendlyName": "Crown Firm List",
+    "shortenedFriendlyName": "Crown Firm List",
+    "url": "",
+    "jurisdictions": ["Crown"],
+    "restrictedProvenances":  []
+  },
+  "MAGS_PUBLIC_LIST": {
+    "friendlyName": "Magistrate Public List",
+    "shortenedFriendlyName": "Magistrates Public List",
+    "url": "",
+    "jurisdictions": ["Crown"],
+    "restrictedProvenances":  []
+  },
+  "MAGS_STANDARD_LIST": {
+    "friendlyName": "Magistrate Standard List",
+    "shortenedFriendlyName": "Magistrates Standard List",
```

Comment:
Same as above, the 2 names are different

Reply:
Now updated